### DISCUSSION TITLE
Remove Data pack Not working

### DISCUSSION THREAD

**Discussion by HoYangE** [community/community#23519] [New to GitHub]
<p>I’m using lfs to pay $5. I deleted the repository to delete the payment, but it still looks like I’m using it. How can I stop paying in a situation that looks like a picture?<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/3/7/37e9f085fe4b0a0a311c003e97f00a74e1cfc09e.jpeg" data-download-href="/uploads/short-url/7YDtocnCX5FD0kaMuhZwZmdPxCK.jpeg?dl=1" title="캡처" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/44865616/181077503-0a65e8ff-ccc9-41dc-a8df-be84450a03e8.jpeg" alt="캡처" data-base62-sha1="7YDtocnCX5FD0kaMuhZwZmdPxCK" width="690" height="213" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/3/7/37e9f085fe4b0a0a311c003e97f00a74e1cfc09e_2_690x213.jpeg, https://aws1.discourse-cdn.com/github/optimized/3X/3/7/37e9f085fe4b0a0a311c003e97f00a74e1cfc09e_2_1035x319.jpeg 1.5x, https://aws1.discourse-cdn.com/github/original/3X/3/7/37e9f085fe4b0a0a311c003e97f00a74e1cfc09e.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/3/7/37e9f085fe4b0a0a311c003e97f00a74e1cfc09e_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">캡처</span><span class="informations">1130×349 19.9 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by queenofcorgis** [comment, marked as answer]
<p>Hi <a class="mention" href="/hoyange">@hoyange</a> - apologies for the delay, this post was in our uncategorized so it received less visibility than it would if it were assigned to a category. In this instance, as this is specific to your account - I’d recommend reaching out to the folks on our <a href="https://support.github.com/contact">Support</a> team as they’ll better be able to assist here :slight_smile:</p>
